### DISCUSSION TITLE
What terminal are you using for examples

### DISCUSSION THREAD

**Discussion by jason-kerney** [spectreconsole/spectre.console#304] [Q&A] [answered]
The screen shot for progress bars (https://spectresystems.github.io/spectre.console/assets/images/progress.png) is really pretty. Mine look nothing like them. What terminal was used to get it to look like that?

**Comment by patriksvensson** [comment, marked as answer]
@jason-kerney That is Windows Terminal. My profile can be found here: https://github.com/patriksvensson/machine/blob/master/config/windows_terminal.json
